nix/k8s-tests/src/k8s_tests_tester.py

```
@@ -1200,6 +1218,469 @@ def _test_s3_storage(self, deployment_config: dict) -> TestResult:
+                f"{base_url}/{narinfo_hash}.narinfo", timeout=HTTP_TIMEOUT
+            )
+            if resp.status_code != 200:
+                return False
+            url_match = re.search(r"^URL:\s*(.+)$", resp.text, re.MULTILINE)
+            if not url_match:
+                return False
+            nar = requests.get(
+                f"{base_url}/{url_match.group(1).strip()}", timeout=HTTP_TIMEOUT
+            )
+            return nar.status_code == 200 and len(nar.content) > 0
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)